Automattic/vip-go-mu-plugins
# Async Cmd Scheduler: implement log2logstash and a Slack alert for audit and debugging purposes

## Description

We were missing the audit/logging for async scheduled commands, which was not great for the debugging.

This PR aims to address the shortcoming by sending a message to Slack audit channel when the command started and when it finished running.

If the command errored (non-0 exit code or something is in STDERR) a message with details gets logged to Logstash.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

It's kind of tricky to test since Alerts service is blocked on dev sandboxes.

1. Apply the PR
2. `wp shell`
3.  Run ` \Automattic\VIP\Commands\Async_Scheduler_Command::runner( 'nonexistent command' );`
4. Watch the warning printed. It should be `The scheduled command \`nonexistent command\` has non-zero exit code or non-empty STDERR`
5.  Run any existing command and verify there are no warnings are printed.
